portal-web/test/functional/com/liferay/portalweb/macros/KBArticle.macro

```diff
@@ -1125,6 +1173,18 @@ definition {
 			locator1 = "Header#H4_TITLE");
 	}
 
+	macro viewArticleExpirationStatus {
+		if (isSet(kbExpirationStatus)) {
+			AssertTextEquals(
+				key_kbArticleTitle = ${kbArticleTitle},
+				locator1 = "KnowledgeBase#ARTICLE_TABLE_EXPIRATION_STATUS",
+				value1 = ${kbExpirationStatus});
+		}
+		else {
+			AssertElementNotPresent(locator1 = "KnowledgeBase#ARTICLE_TABLE_EXPIRATION_STATUS");
+		}
```

Comment:
This won't work because you still need to pass in key_kbArticleTitle for the locator to be accurate. It's giving a false positive in the test report

Better to split this off into a different macro